packages/web-components/fast-foundation/src/accordion-item/stories/accordion-item.register.ts

```diff
@@ -1,3 +1,5 @@
+import chevronDownIcon from "@fluentui/svg-icons/icons/chevron_down_12_regular.svg";
+import chevronUpIcon from "@fluentui/svg-icons/icons/chevron_up_12_regular.svg";
```

Comment:
@bheston I really want to avoid any dependency - dev or otherwise to Fluent here. It seems like we could have our own test icons to validate but I don't think we should couple ourselves to the Fluent icons for testing. Our consideration here is not _what icon_ or even whether it works with the test styling here, simply that it can be passed. Can we create local test icons rather than take a dependency on Fluent?

Reply:
Initially I had a local copy of all icons but I thought copying in a bunch of svg, even if just a few icons, wasn't the best solution. I can go back to that if you don't mind, but I do like the idea of "outsourcing" the icons. I feel like the Fluent icons in particular are good and tangentially related to FAST, but also mostly conveniently named as "Fluent". Updates on the Fluent side would not make a material difference in the testing of the components or the stories.

Reply:
I figured out how to move this, though this model requires people to find the _right_ icons when they need to add anything new.